core/kazoo_apps/priv/couchdb/account/users.json

```diff
@@ -10,25 +10,115 @@
+            "map": [
+                "function(doc) {",
+                "  if (doc.pvt_type != 'user' || doc.pvt_deleted) return;",
+                "  var features = [];",
+                "  if (doc.smartpbx)",
+                "    for (var feature in doc.smartpbx)",
+                "      if(doc.smartpbx[feature].enabled) features.push(feature);",
+                "  if (doc.hotdesk && doc.hotdesk.enabled) features.push('hotdesk');",
+                "  if (doc.call_forward && doc.call_forward.enabled) features.push('call_forward');",
+                "  if (doc.call_forward && doc.call_forward.failover && !doc.call_forward.enabled) features.push('call_forward_failover');",
+                "  if (doc.caller_id && doc.caller_id.external && doc.caller_id.external.number) features.push('caller_id');",
+                "  if (doc.vm_to_email_enabled) features.push('vm_to_email');",
+                "  if (doc.music_on_hold && doc.music_on_hold.media_id) features.push('music_on_hold');",
+                "  if (doc.do_not_disturb && doc.do_not_disturb.enabled) features.push('do_not_disturb');",
+                "  emit(doc.last_name + ' ' + doc.first_name, {",
+                "    'id': doc._id,",
+                "    'features': features,",
+                "    'username': doc.username,",
+                "    'email': doc.email,",
+                "    'first_name': doc.first_name,",
+                "    'last_name': doc.last_name,",
+                "    'priv_level': doc.priv_level,",
+                "    'feature_level': doc.feature_level,",
+                "    'presence_id': doc.presence_id,",
+                "    'timezone': doc.timezone,",
+                "    'call_recording': doc.call_recording,",
+                "    'service': doc.service,",
+                "    'ctu': doc.ctu",
```

Comment:
isn't ctu inside doc.smartpbx[ctu] ?

Reply:
idk, in the ticket it is on the root. it is not mentioning smartpbx.